iroha/iroha.py

```diff
@@ -197,7 +255,8 @@ def query(self, name, counter=1, creator_account=None,
               created_time=None, page_size=None,
               first_tx_hash=None, first_tx_time=None,
               last_tx_time=None, first_tx_height=None,
-              last_tx_height=None, **kwargs):
+              last_tx_height=None, ordering_sequence=None,
+              **kwargs):
```

Comment:
ordering sequence should be specified before first_tx_time I think (to make it same as in other libraries and Iroha)

Reply:
You are right, thanks.

Reply:
Before first_Tx_time and after first_tx_hash: https://iroha.readthedocs.io/en/develop/develop/api/queries.html

Reply:
Thanks, corrected in https://github.com/hyperledger/iroha-python/pull/83/commits/943998e2a90bd48d5378e9ec4d9ea2d9fe75cebc

Resolved